### ISSUE TITLE
ShipSaveSplicer installed into wrong directory

### ISSUE THREAD

**Issue by linuxgurugamer** [KSP-CKAN/CKAN#3187]
Background
----------
* Windows 10
* CKAN Version: 1.28.0
* KSP Version: 

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?**
no

Problem
-------
ShipSaveSplicer is being installed into a directory called "veSplicer"
I verified that the file on Spacedock is correct, and the netkan is pretty simple.

**Comment by DasSkelett** [comment]
The stripping happens exactly here:
https://github.com/KSP-CKAN/CKAN/blob/a7ad4af8078082afff9d7f4631e21d2682ed167f/Core/Types/ModuleInstallDescriptor.cs#L508-L516

**Comment by DasSkelett** [comment]
Appending a `/` to the `ReservedPaths` (and removing the `+ 1` from the `Substring`) does indeed fix this bug, and still keeps the functionality to strip the reserved paths for stanzas like this:
```
"install": [
        {
            "find": "GameData",
            "install_to": "GameData"
        }
    ],
```

@HebaruSan it looks like this is a two line fix, kinda fitting into #3180. Do you want to add it there, or should I do a separate PR?

**Comment by HebaruSan** [comment]
Nice, I'll add it to #3180.
